app/src/behaviors/behavior_tap_dance.c

```diff
@@ -156,6 +157,12 @@ static int on_tap_dance_binding_pressed(struct zmk_behavior_binding *binding,
     // value, invoke the last binding available.
     if (tap_dance->counter < cfg->behavior_count) {
         tap_dance->counter++;
+        if (cfg->trigger_each_binding) {
+            behavior_keymap_binding_pressed(&tap_dance->config->behaviors[tap_dance->counter - 1],
```

Comment:
FYI you should use `zmk_behavior_invoke_binding` instead, see https://github.com/zmkfirmware/zmk/commit/9e36ebd52587c0364562057466a458fdcfcdc685#diff-7eeba6e44a877bcea40b140ff0aaed1559f31fdea34ba7f71f0223b84c68e881

Reply:
Fixed!